Source/HTTP/compression.cpp

```diff
@@ -3,7 +3,7 @@
 #include "pch.h"
 
 #if !HC_NOZLIB
-#if HC_PLATFORM == HC_PLATFORM_WIN32 || HC_PLATFORM == HC_PLATFORM_GDK
+#if HC_PLATFORM == HC_PLATFORM_WIN32 || HC_PLATFORM == HC_PLATFORM_GDK || HC_PLATFORM == HC_PLATFORM_NINTENDO_SWITCH
```

Comment:
Will this break Switch builds until zlib build material has been added to the LHC.Switch repo?

Reply:
It should not break the Switch build; this will only expose the Compression Functionality to Switch, the actual calls to all of this are in the PlayFab.C PR. We should be fine for now.